istack-commons/pom.xml

```diff
@@ -119,7 +119,7 @@
             <dependency>
                 <groupId>org.glassfish.jaxb</groupId>
                 <artifactId>codemodel</artifactId>
-                <version>2.3.2</version>
+                <version>2.3.2-1</version>
```

Comment:
This is breaking SemVer 2.0:
```
<valid semver> ::= <version core>
                 | <version core> "-" <pre-release>
                 | <version core> "+" <build>
                 | <version core> "-" <pre-release> "+" <build>
```
1. pre-release identifier shall start with non digit character
2. pre release version < release version
https://semver.org/

Reply:
After some discussion looks like there is no better way in maven.

Reply:
maven does not use semver, to verify:

```
ljungman-mac:jaxws-ri lukas$ java -jar /Users/lukas/java/apache-maven-3.6.0/lib/maven-artifact-3.6.0.jar 2.3.2 2.3.2-1
Display parameters as parsed by Maven (in canonical form) and comparison result:
1. 2.3.2 == 2.3.2
   2.3.2 < 2.3.2-1
2. 2.3.2-1 == 2.3.2-1
```